src/db/seeds.rb

```diff
@@ -6,4 +6,8 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 # TODO: change email address
-User.create(username: 'admin', email: 'admin@example.com', password: 'admin', password_confirmation: 'admin', role: 'admin')
+User.create(username: 'admin', email: 'admin@example.com', password: 'admin', password_confirmation: 'admin',role: 'admin')
+teacher = User.create(username: 'teacher', email: 'teacher@example.com', password: 'teacher', password_confirmation: 'teacher', role: 'teacher')
+
+course = Course.create! name: "Computer Course", teacher_id: teacher.id
+Quiz.create! title: "Computer Science ", course_id: course.id
```

Comment:
Why should we add  these things to a production environment? The admin user is intended to bootstrap the whole sys like a root user in unix.

The rest should go into test/fixtures if you need them for your (unit) tests.